static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -109,6 +109,10 @@
     .adverts__row > * + *::before {
         content: none;
     }
+
+    .adverts__badge__help {
```

Comment:
You know what I mean...badge is pretty broad term, so do we want to take it for commercial? But I suppose why not.